USERS.md

```diff
@@ -945,3 +945,8 @@ Users (Alphabetically)
       D: Zero Hash is using Cilium as CNI for networking, security and monitoring features for Kubernetes clusters
       U: CNI/ENI Networking, Network policies, Hubble
       Q: @eugenestarchenko
+
+    * N: Sagar Mujumale
```

Comment:
This should be the name of the company not yourself. Please also put it in alphabetical order